deepchecks/vision/batch_wrapper.py

```diff
@@ -94,6 +96,23 @@ def __len__(self):
         dataset = self._context.get_data_by_kind(self._dataset_kind)
         return len(list(dataset.data_loader.batch_sampler)[self.batch_index])
 
+    def vision_properties(self, raw_data: List, properties_list: List[Dict], input_type=PropertiesInputType):
```

Comment:
```suggestion
    def vision_properties(self, raw_data: List, properties_list: List[Dict], input_type: PropertiesInputType):
```
I guess you meant this?